packages/sliderInput/components/SliderInput.tsx

```diff
@@ -0,0 +1,207 @@
+import * as React from "react";
+import { cx } from "emotion";
+import {
+  flex,
+  flexItem,
+  visuallyHidden,
+  display,
+  tintContent,
+  tintContentSecondary,
+  vAlignChildren
+} from "../../shared/styles/styleUtils";
+
+import {
+  themeError,
+  themeSuccess,
+  themeBgPrimary
+} from "../../design-tokens/build/js/designTokens";
+import { InputAppearance } from "../../shared/types/inputAppearance";
+import FormFieldWrapper from "../../shared/components/FormFieldWrapper";
+import { SystemIcons } from "../../icons/dist/system-icons-enum";
+import { Icon } from "../../icon";
+import {
+  checkContainer,
+  slider,
+  sliderRound,
+  sliderContainer,
+  sliderInputApperances,
+  sliderInputLabel,
+  sliderInputFeedbackText,
+  bruteForceKillLabelMargin
+} from "../style";
+
+export interface SliderInputProps extends React.HTMLProps<HTMLInputElement> {
+  /**
+   * Sets the current appearance of the input component. This defaults to InputAppearance.Standard, but supports `InputAppearance.Error` & `InputAppearance.Success` appearances as well.
+   */
+  appearance?: InputAppearance;
+  /**
+   * Whether or not the input is checked
+   */
+  checked?: boolean;
+  /**
+   * Unique identifier used for the input element
+   */
+  id: string;
+  /**
+   * The text or node content that appears next to the input
+   */
+  inputLabel: React.ReactNode | string;
+  /**
+   * Defaults to `true`, but can be set to `false` to visibly hide the content passed to `inputLabel`. The `inputLabel` should still be set even when hidden for accessibility support.
+   */
+  showInputLabel?: boolean;
+  /**
+   * The value being toggled
+   */
+  value?: string;
+  /**
+   * How the text content vertically aligns with the input
+   */
+  vertAlign?: "center" | "top";
+  /**
+   * hintContent is text or a ReactNode that is displayed directly under the input with additional information about the expected input.
+   */
+  hintContent?: React.ReactNode;
+  /**
+   * Sets the contents for validation errors. This will be displayed below the input element. Errors are only visible when the `TextInput` appearance is also set to `TextInputAppearance.Error`.
+   */
+  errors?: React.ReactNode[];
+}
+
+const SliderInput: React.FC<React.PropsWithRef<SliderInputProps>> = props => {
+  const {
+    appearance,
+    children,
+    disabled,
+    hintContent,
+    id,
+    inputLabel,
+    showInputLabel,
+    vertAlign,
+    checked,
+    value,
+    errors,
+    onBlur,
+    onFocus,
+    ...other
+  } = props;
+  const inputType = "Slider";
+  const inputDataCy = [
+    `${inputType}Input-input`,
+    ...(checked ? [`${inputType}Input-input.checked`] : []),
+    ...(appearance && appearance !== InputAppearance.Standard
+      ? [`${inputType}Input-input.${appearance}`]
```

Comment:
i think at some point in time we decided not to make as big of a story out of those helper-classes and just went with not hiding the `Standard`-case as to reduce the noise.  
that's optional of course.